index.js

```diff
@@ -1,5 +1,8 @@
 //
 // Setup and Execution
+
+const { DateTime } = require('luxon');
```

Comment:
Can't we handle this with built in DateTime?

Reply:
the formatting is easier with Luxon, kind of like using chalk for the coloring of text

Reply:
What does it look like when you use the built-in one?